content/en/docs/cluster-administration/persistent-volume-and-storage-class.md

```diff
@@ -77,18 +74,18 @@ QingCloud CSI is a CSI plugin on Kubernetes for the storage service of QingCloud
 
 #### Settings
 
-![storage-volume-qingcloud](/images/docs/cluster-administration/persistent-volume-and-storage-class/storage-volume-qingcloud.jpg)
+![storage-volume-qingcloud](/images/docs/cluster-administration/persistent-volume-and-storage-class/storage-volume-qingcloud.png)
 
 | Property | Description |
 | :---- | :---- |
-| type     | On the QingCloud platform, 0 represents high performance volumes. 2 represents high capacity volumes. 3 represents super high performance volumes. 5 represents Enterprise Server SAN. 6 represents NeonSan HDD. 100 represents standard volumes. 200 represents enterprise SSD. |
+| type     | On the QingCloud platform, 0 represents high performance volumes. 2 represents high capacity volumes. 3 represents super high performance volumes. 5 represents Enterprise Server SAN. 100 represents standard volumes. 200 represents enterprise SSD. |
```

Comment:
Please confirm the deletion here.

Reply:
Confirmed. The original sentence should be removed.